mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -3,83 +3,83 @@ CLASS net/minecraft/unmapped/C_lscptbhn net/minecraft/client/option/GameOptions
 	FIELD f_becjowrb hideServerAddress Z
 	FIELD f_biydppgq chatLinksPrompt Z
 	FIELD f_bjnodgyr difficulty Lnet/minecraft/unmapped/C_mpbjgxic;
-	FIELD f_bjoysnee keysAll [Lnet/minecraft/unmapped/C_sygikzty;
+	FIELD f_bjoysnee allKeys [Lnet/minecraft/unmapped/C_sygikzty;
 		COMMENT An array of all key bindings.
 		COMMENT
 		COMMENT <p>Key bindings in this array are shown and can be configured in
 		COMMENT {@linkplain net.minecraft.client.gui.screen.option.ControlsOptionsScreen
 		COMMENT the controls options screen}.
-	FIELD f_bwxrmuqn sprintToggled Z
+	FIELD f_bwxrmuqn toggleableSprint Z
 	FIELD f_cfdciozt enabledPlayerModelParts Ljava/util/Set;
-	FIELD f_cfrcggxy autoSuggestions Z
+	FIELD f_cfrcggxy commandSuggestions Z
 	FIELD f_cgvttuzt language Ljava/lang/String;
-	FIELD f_ckfylycx chatHeightFocused D
+	FIELD f_ckfylycx focusedChatHeight D
 	FIELD f_cnqkqzfz entityDistanceScaling F
 	FIELD f_cpeiozpi textBackgroundOpacity D
-	FIELD f_cxkxaksc ao Lnet/minecraft/unmapped/C_pgtmqhfp;
+	FIELD f_cxkxaksc smoothLightning Lnet/minecraft/unmapped/C_pgtmqhfp;
 	FIELD f_daisjzsp heldItemTooltips Z
-	FIELD f_ddtthvbj keyFullscreen Lnet/minecraft/unmapped/C_sygikzty;
+	FIELD f_ddtthvbj fullscreenKey Lnet/minecraft/unmapped/C_sygikzty;
 		COMMENT A key binding for toggling fullscreen.
 		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_F11 the F11 key} by default.
 	FIELD f_dhaombld skipMultiplayerWarning Z
 	FIELD f_dntllpjh chatScale D
 	FIELD f_docivyey autoJump Z
-	FIELD f_dognqaxv touchscreen Z
+	FIELD f_dognqaxv touchscreenMode Z
```

Comment:
`isTouchscreen` or `touchscreen` make more sense IMO. `touchscreenMode` sounds like an enum.

Reply:
`is` is discouraged on cases like that, so it can't be that; Fair enough, I'll revert that one